build/verify-tools.ts

```diff
@@ -0,0 +1,80 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+const hasha = require('hasha');
+const mkdirp = require('mkdirp');
+const request = require('request');
+const fs = require('fs-extra');
+const path = require("path");
+const progress = require('request-progress');
+const configData = require('../src/tools.json');
+const os = require('os');
+
+async function verifyTools() {
+  for (const key in configData) {
+    for (const OS in configData[key].platform) {
+      const targetFolder =  path.resolve(os.tmpdir(), OS);
+      console.log(`[INFO] ${configData[key].platform[OS].dlFileName} is not downloaded yet`);
+      await downloadFileAndCreateSha256(targetFolder, configData[key].platform[OS].dlFileName, configData[key].platform[OS].url, configData[key].platform[OS].sha256sum);
+    }
+  }
+}
+
+async function downloadFileAndCreateSha256(targetFolder, fileName, reqURL, sha256sum) {
+  if (!fs.existsSync(targetFolder)) {
+    await mkdirp.sync(targetFolder);
+  }
+  const currentFile = path.join(targetFolder, fileName);
+  await downloadFile(reqURL, currentFile);
+  const currentSHA256 = await getSHA256(currentFile);
+  if (currentSHA256 === sha256sum) {
+    console.log( `[INFO] ${currentFile} is downloaded and sha256 is correct`);
+  } else {
+    throw console.log(`[Error] ${currentFile} is downloaded and sha256 is not correct`);
```

Comment:
This will produce 
```
npm run verify

> vscode-openshift-connector@0.0.22 verify c:\p\vscost
> node ./build/verify-tools.ts

[INFO] odo-windows-amd64.exe.gz is not downloaded yet
[INFO] C:\Users\dgolo\AppData\Local\Temp\win32\odo-windows-amd64.exe.gz download started from https://github.com/redhat-developer/odo/releases/download/v0.0.20/odo-windows-amd64.exe.gz
0%
1%
9%
50%
99%
100%
[Error] C:\Users\dgolo\AppData\Local\Temp\win32\odo-windows-amd64.exe.gz is downloaded and sha256 is not correct
(node:1988) UnhandledPromiseRejectionWarning: undefined
```

Instead of use console.log, just use ```throw Error('error message')```.